snippets/language-html.cson

```diff
@@ -121,7 +121,7 @@
     'body': '<footer>$1</footer>$0'
   'Form':
     'prefix': 'form'
-    'body': '<form id="${1:form_id}" action="${2:index.html}" method="${3:post}">\n\t$4\n</form>'
+    'body': '<form class="${1:form_class}" action="${2:index.html}" method="${3:post}">\n\t$4\n</form>'
```

Comment:
Or actually, looks like most have just a cursor holder, might be nice to standardize them all since they are all getting changed.